src/arvparams.h.in

```diff
@@ -0,0 +1,36 @@
+/* Aravis - Digital camera library
+ *
+ * Copyright © 2009-2022 Emmanuel Pacaud
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General
+ * Public License along with this library; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ *
+ * Author: Emmanuel Pacaud <emmanuel@gnome.org>
+ */
+
+#ifndef ARV_PARAMS_H
+#define ARV_PARAMS_H
+
+/**
+ * ARV_GV_STREAM_NUM_BUFFERS
+ *
+ * Number of buffers used to receive GVSP packets
+ *
+ * Since: 0.8.21
+ */
+
+#mesondefine ARV_GV_STREAM_NUM_BUFFERS
+
+#endif
```

Comment:
A dedicated header is a bit overkill, but let's keep it. Rename it to arvparamsprivate.h please, and don't install it. I don't want to make this define public, at least for now.